README.md

```diff
@@ -17,14 +27,16 @@ python sd_card_image_generator.py --sd-card-path /path/to/sd/card --zip-file-pat
 python sd_card_image_generator.py --help
 ````
 
-### reconstruction_dummy.py
+### trace_reconstruction.py
+#### Description:
+This script offers a convenient and efficient way to reconstruct data given a SD-Card image and a folder of CSV trace files based on a user-specified timestamp.
 #### Execute from command line:
 ````
-python reconstruction_dummy.py --reconstruction-path E:\\Vmshare --csv-path E:\\Vmshare\\CSV --image-name sdCard
+python trace_reconstruction.py --image-path E:\Vmshare\sdCard.img --csv-path E:\Vmshare\CSV --date 18.05.2023 --time 14:22:02
```

Comment:
Use a generic description of arguments instead of examples

Reply:
resolved in [41036d9](https://github.com/INFB-Projekt/flashspeicher-rekonstruktion/pull/22/commits/41036d9f990fbee8a86b498aa857c4556409fb70)